ArduCopter/config.h

```diff
@@ -133,6 +133,10 @@
  # define EKF_ORIGIN_MAX_ALT_KM         50   // EKF origin and home must be within 50km vertically
 #endif
 
+#ifndef EKF_LPF_CUTOFF
+# define EKF_LPF_CUTOFF     5.0f    // frequency cutoff of EKF variance filters
```

Comment:
Have you tried a smaller value? When tuning I would err on the side of the smallest value that prevents failsafe due to transient errors. How early we want to fall back into a non position mode, eg from LOITER to ALTITUDE is contextual. For example two cases that would be in tension wrt setting this value are a) flight in LOITER close to pilot vs up and away flight in AUTO at a distance from the pilot.

Reply:
So you think I might need a more aggressive cutoff to avoid transient errors?

Reply:
I guess it really needs a parameter

Reply:
I have added a parameter. Still needs flight testing!

Reply:
Sorry! When I saw the value of 5.0 I was thinking in terms of a time constant, but see now you are using it as a frequency.